ansible/README.md

```diff
@@ -0,0 +1,29 @@
+ansible
+===
+
+An Ansible role to install Ansible.
+
+Requirements
+------------
+
+Assumes RedHat OS family and access to the Internet (or access to local Pypi servers)
+
+Role Variables
+--------------
+
+    version: 2.6.5
```

Comment:
I'd prefix this to `ansible_version`, variables can be set in a group_vars file if they are prefixed to the role. If they are named generically they must be passed directly on the role.